src/clients/rc_client_pgsql.erl

```diff
@@ -0,0 +1,12 @@
+-module (rc_client_pgsql).
+
+-export ([create_db/1, insert/1, select/1]).
+
+create_db(Pid) ->
```

Comment:
I think it would be probably more convenient for quick tests to have a single function that creates, inserts and selects instead of separate functions. The same for redis